README.md

```diff
@@ -53,6 +54,19 @@ Please see [CONTRIBUTING](https://github.com/duyenle1312/testing-library/blob/ma
 - [Unity](www.throwtheswitch.org/unity)
 - [utest](https://github.com/evolutional/utest)
 
+## CPP ##
```

Comment:
Could you help me remove the ## next to CPP? By the way, isn't C++ a language name instead of Cpp as a file extension?